.github/actions/prepare-install/action.yml

```diff
@@ -31,7 +31,9 @@ runs:
     - uses: actions/cache@v3
       id: yarn-cache
       with:
-        path: ${{ steps.yarn-cache-dir-path.outputs.dir }}
+        path: |
+          ${{ steps.yarn-cache-dir-path.outputs.dir }}
```

Comment:
This is essentially a subpath of `~/.cache`. I don't know if there would be any adverse effects if we simply cache the entire `~/.cache`, or if I should constrain it to certain subpaths.